Gemfile

```diff
@@ -32,6 +32,7 @@ group :development, :test do
 end
 
 group :development do
+  gem "pry"
```

Comment:
I've been using `binding.break` from Rails 7 instead of Pry lately https://www.mintbit.com/blog/byebug-replaced-with-debug-rails7

Is Pry better?

Reply:
The one case where both kind of fail in my testing is when combined with foreman; it swallows the repl somehow and I can't see what I'm typing. In that case I fall back to running `rails s` manually